lib/ffmpeg/qsv/vpp.py

```diff
@@ -71,3 +71,6 @@ def gen_vpp_opts(self):
       )
 
     return vpfilter
+
+  def hwinit_format(self):
+    return f"qsv=qsv,child_device={self.hwdevice}"
```

Comment:
This doesn't seem right since `self.hwdevice = "hw_any,child_device={self.renderDevice}"` in `before()` method...

Which would result something like this: `"qsv=qsv,child_device=hw_any,child_device=/dev/dri/renderD128"`

Reply:
The default impl mode is auto, and the device is specified via child_device key.

Reply:
`child_device` was in the first commit for QSV, see 59e7361cc791e5103be1712dc59a2055f118d0da, and was used by lots of users, so maybe we can't change it. 
the whole format for device initialization is `type=name:device,key1=xxx,key2=xxx`, for qsv, device is mapped to implementation (auto, hw_any, hw1, ... etc), however hw1, hw2 etc on Linux is not supported in MediaSDK,  so the key of `child_device` is used to specify the underlying child device.

Reply:
Ok, perhaps it's best to use https://github.com/intel/vaapi-fits/pull/455#discussion_r1007496961 suggestion.  And just override as needed.